app/src/main/res/layout/fragment_team.xml

```diff
@@ -62,7 +62,7 @@
                 android:layout_height="wrap_content"
                 android:gravity="center"
                 android:padding="@dimen/padding_small"
-                android:text="Type"
+                android:text="@string/typename"
```

Comment:
I also don't think string file changes are necessary for this pr. so revert these changes on the strings files